azuki-vdm/src/main/kotlin/com/anaplan/engineering/azuki/vdm/animation/VdmEacAnimator.kt

```diff
@@ -21,7 +21,7 @@ class VdmEacAnimator(private val animationModule: AnimationModule) {
 
     private fun createAnimationFile(animationModule: AnimationModule): VdmFile {
         return VdmFile("Animation",
-            """
+"""-- @Warning(5000) suppress warnings for unused definitions
```

Comment:
Unfortunately this annotation is not currently honoured by Overture, although it is honoured by VDMJ. Still, there is no harm adding it now.